chainer/functions/loss/mean_absolute_error.py

```diff
@@ -53,5 +59,36 @@ def mean_absolute_error(x0, x1):
             A variable holding an array representing the mean absolute
             error of two inputs.
 
+    .. admonition:: Example
+
+    1D array examples
```

Comment:
`.. admonition:: Example` starts a block.  See `chainer/functions/loss/huber_loss.py` for example.

Reply:
Should I change anything?

Reply:
Could you indent the example by 4 more spaces?